src/components/pages/index.js

```diff
@@ -1,47 +1,27 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Markdown from '../markdown.js';
 import urls from '../../utils/urls.js';
 import SearchBar from '../search-bar.js';
-import { useSettings } from '../settings-context.js';
 import { useGlobalAlerts } from '../../ui/global-alerts.js';
 
-function Index({ PageTemplate, HomePageLogo, SiteLogo }) {
-  const settings = useSettings();
-
+function Index({ HomePageTemplate }) {
   const handleSearch = searchText => {
     window.location = urls.getSearchUrl(searchText);
   };
 
   const alerts = useGlobalAlerts();
 
   return (
-    <PageTemplate alerts={alerts} fullScreen>
+    <HomePageTemplate alerts={alerts} fullScreen>
```

Comment:
I think as there is only one homepage, we do not need the distinction fullscreen=true/false ?

Reply:
Was thinking the same, but if no `HomePageTemplate` is set, we default to `DefaultPageTemplate` which requires `fullScreen`.
However we will pass in a HomePageTemplate anyway from both projects, so then I guess we could just simplify and drop `fullScreen`

Reply:
Scratch that, I got it wrong 🤦‍♀️ Will remove `fullScreen`

Reply:
Ah, I see. Then I'll leave that decision to your taste :-)

Reply:
Ok, too late :laughing:

Reply:
😅 thanks! Updated though!